tests/test_e2e.py

```diff
@@ -130,49 +107,40 @@ def test_tensorrtllm_e2e(self, protocol, setup_and_teardown):
         ],
     )
     @pytest.mark.timeout(900)
-    def test_vllm_e2e(self, protocol, setup_and_teardown):
+    def test_vllm_e2e(self, protocol):
         # NOTE: VLLM test models will be passed by the testing infrastructure.
         # Only a single model will be passed per test to enable tests to run concurrently.
         model = os.environ.get("VLLM_MODEL")
         assert model is not None, "VLLM_MODEL env var must be set!"
         self._import(model)
-        pid = utils.run_server()
-        setup_and_teardown.pid = pid
-        # vLLM will download the model on the fly, so give it a big timeout
-        # TODO: Consider one of the following
-        # (a) Pre-download and mount larger models in test environment
-        # (b) Download model from HF for vLLM at import step to remove burden
-        #     from server startup step.
-        utils.wait_for_server_ready(timeout=600)
-
-        self._infer(model, prompt=PROMPT, protocol=protocol)
-        self._profile(model, backend="vllm")
+        with utils.MockServer(timeout=600) as _:
```

Comment:
I believe you can remove all of the `as _` if not using them. Correct me if wrong though.

Reply:
Thanks for the catch! I got so used to writing `with ____ as ____` that I never realized you could omit the latter.